base/strings/basic.jl

```diff
@@ -48,6 +48,8 @@ sizeof(s::AbstractString) = error("type $(typeof(s)) has no canonical binary rep
 eltype{T<:AbstractString}(::Type{T}) = Char
 
 (*)(s1::AbstractString, ss::AbstractString...) = string(s1, ss...)
+(.*){T<:AbstractString}(v::Vector{T},s::T) = [i*s for i in v]
```

Comment:
Why does `s` need to be of the same type as the elements of `v`?  Probably should just be `s::AbstractString`